rules/CodingStyle/Application/UseImportsAdder.php

```diff
@@ -31,33 +34,33 @@ public function __construct(
     /**
      * @param Stmt[] $stmts
      * @param array<FullyQualifiedObjectType|AliasedObjectType> $useImportTypes
-     * @param array<FullyQualifiedObjectType|AliasedObjectType> $constantUseImportTypes
      * @param array<FullyQualifiedObjectType|AliasedObjectType> $functionUseImportTypes
+     * @param array<FullyQualifiedObjectType|AliasedObjectType> $constantUseImportTypes
      * @return Stmt[]
      */
     public function addImportsToStmts(
         FileWithoutNamespace $fileWithoutNamespace,
         array $stmts,
         array $useImportTypes,
-        array $constantUseImportTypes,
-        array $functionUseImportTypes
+        array $functionUseImportTypes,
+        array $constantUseImportTypes
     ): array {
         $usedImports = $this->usedImportsResolver->resolveForStmts($stmts);
         $existingUseImportTypes = $usedImports->getUseImports();
-        $existingConstantUseImports = $usedImports->getConstantImports();
-        $existingFunctionUseImports = $usedImports->getFunctionImports();
+        $existingFunctionUseImportTypes = $usedImports->getFunctionImports();
+        $existingConstantUseImportTypes = $usedImports->getConstantImports();
 
-        $useImportTypes = $this->diffFullyQualifiedObjectTypes($useImportTypes, $existingUseImportTypes);
-        $constantUseImportTypes = $this->diffFullyQualifiedObjectTypes(
-            $constantUseImportTypes,
-            $existingConstantUseImports
-        );
-        $functionUseImportTypes = $this->diffFullyQualifiedObjectTypes(
+        $newUseImportTypes = $this->diffFullyQualifiedObjectTypes($useImportTypes, $existingUseImportTypes);
+        $newFunctionUseImportTypes = $this->diffFullyQualifiedObjectTypes(
```

Comment:
PSR-12 and PER standards dictate that the order should be normal, function, const, so we change the order of these parameters to reflect this order